### ISSUE TITLE
[PrettyPrint] Error parsing regular expression: unterminated parenthetical

### ISSUE THREAD

**Issue by digitarald** [devtools-html/debugger.html#3702]
Migrated from https://bugzilla.mozilla.org/show_bug.cgi?id=1370073

```
STR
1. Open http://www.getginger.jp/
2. Attempt to pretty print platform.twitter.com widgets.js

AR:
Nothing happens:

Error in browser console:
"Error in worker resource://devtools/client/debugger/new/pretty-print-worker.js"  debugger.js:32612:8
DataCloneError: The object could not be cloned.  pretty-print-worker.js:5930

ER
Perform pretty print
```

Confirmed on Nightly:

```
SyntaxError: Error parsing regular expression: unterminated parenthetical (8:23864) pretty-print-worker.js:1280:16
"Error in worker resource://devtools/client/debugger/new/pretty-print-worker.js" debugger.js:32725:8
start/this.worker.onerror
```

**Comment by assafdd** [comment]
Seems duplicated with this one 
https://github.com/devtools-html/debugger.html/issues/4619

**Comment by digitarald** [comment]
Closed via #4619
